rpm/sailfish-office.spec

```diff
@@ -15,6 +15,7 @@ BuildRequires: poppler-qt5-devel poppler-qt5 poppler-devel poppler
 BuildRequires: mapplauncherd-qt5-devel
 BuildRequires: cmake
 BuildRequires: qt5-qttools-linguist
+BuildRequires: libicu
```

Comment:
Done

Reply:
Since using pkg_config, something like `pkg_check_modules(ICU icu-i18n REQUIRED)` should be added in the `CMakeLists.txt` in my opinion. Then add the include and libs to the required variables. It is compiling by luck that unicode/ucsdet.h is in the path and that libicu is already in memory, because the plugin is not linked against.

Reply:
Something like:
```
find_package(PkgConfig REQUIRED)
pkg_check_modules(SAILFISHSILICA sailfishsilica REQUIRED)
pkg_check_modules(ICU icu-i18n REQUIRED)

include_directories(
    ${SAILFISHSILICA_INCLUDE_DIRS}
    ${ICU_INCLUDE_DIRS}
)

set(plugin_SRCS
    fileinfo.cpp
    plaintextmodel.cpp
    sailfishofficeplugin.cpp
)

add_library(sailfishofficeplugin MODULE ${plugin_SRCS})
qt5_use_modules(sailfishofficeplugin Widgets Quick)
target_link_libraries(sailfishofficeplugin stdc++ ${QT_LIBRARIES} ${SAILFISHSILICA_LIBRARIES} ${ICU_LIBRARIES})
```

Reply:
Done

Reply:
Thanks.